libs/sdk-bindings/bindings-android/lib/src/main/kotlin/breez_sdk_notification/job/LnurlPayInfo.kt

```diff
@@ -58,7 +69,7 @@ class LnurlPayInfoJob(
             val response =
                 LnurlPayInfoResponse(
                     request.callbackURL,
-                    nodeState.inboundLiquidityMsats,
+                    maxSendable,
```

Comment:
The maxSendable should also be the max of feeLimitAmountMsat and the nodeState.inboundLiquidityMsats, because maybe it has more inbound already available than the calculated feeLimitAmountMsat

Reply:
So, we should calculate maxSendable as:
`max( min(maxReceivable, maxReceivableThatFallsWithinFeeLimit), inboundLiquidity)`
?

Reply:
Done 85db9cf6bd2e25487bf7fb6d1b86f78cf00962f0